pom.xml

```diff
@@ -383,6 +384,12 @@
 			<version>1.4.1</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.assertj</groupId>
+			<artifactId>assertj-core</artifactId>
+			<version>${assertj.version}</version>

```

Comment:
[It appears to be](https://github.com/devhub-tud/devhub/pull/423/files/dacec5b3708936ff3b79ea44d5fc5a99b799b39d..e9f8e2091d22f38eed830e6f11c896c419fd0552#diff-600376dffeb79835ede4a0b285078036R391)